### DISCUSSION TITLE
请求重试全部失败

### DISCUSSION THREAD

**Discussion by 2622594863** [hect0x7/JMComic-Crawler-Python#268] [Q&A]
jmcomic.jm_exception.RequestRetryAllFailException: 请求重试全部失败: [/setting], ['www.jmapinodeudzn.xyz', 'www.cdn-eldenringproxy.xyz', 'www.cdn-eldenringproxy.me', 'www.cdn-eldenringproxy.vip', 'www.jmapinode.xyz']
[9388] Failed to execute script 'main' due to unhandled exception!

请问我电脑可以使用，换台设备就不行，就很奇怪

**Comment by hect0x7** [comment]
换台设备就不行，啥意思？换啥设备，也是电脑吗

**Comment by hect0x7** [comment]
这个异常是重试全部失败，应该还有每次重试失败原因的日志，也贴出来
